### PR TITLE
Standardize EDC setting keys (#84)

### PR DESCRIPTION
**Pull request by bscholtes1A**
Apply standard naming for all EDC settings: edc.*

### CODE REVIEW

File: core/bootstrap/src/main/java/org/eclipse/dataspaceconnector/system/CoreServicesExtension.java [hunk 1]
```diff
@@ -28,9 +29,15 @@
 
     public static final String FEATURE_HTTP_CLIENT = "dataspaceconnector:http-client";
     public static final String FEATURE_RETRY_POLICY = "dataspaceconnector:retry-policy";
-    private static final String MAX_RETRIES = "dataspaceconnector:core.retry.max-retries";
-    private static final String BACKOFF_MIN_MILLIS = "dataspaceconnector.core.retry.backoff.min";
-    private static final String BACKOFF_MAX_MILLIS = "dataspaceconnector.core.retry.backoff.max";
+
+    @EdcSetting
+    private static final String MAX_RETRIES = "edc.core.retry.max-retries";
```
Comment (jimmarino) [inline core/bootstrap/src/main/java/org/eclipse/dataspaceconnector/system/CoreServicesExtension.java:34]: Hi Benjamin, do you think we should also change max-retries to max.retries for consistency? I don't have any particular preference, I just wanted to raise it as a qyestion

**Review by jimmarino — COMMENTED** [review]
Thanks, Benjamin, looks good to me. I had one minor question, otherwise, let's see what others think and we can merge if there are no issues.

File: core/bootstrap/src/main/java/org/eclipse/dataspaceconnector/system/CoreServicesExtension.java (hunk 1 above)
Comment (bscholtes1A) [reply core/bootstrap/src/main/java/org/eclipse/dataspaceconnector/system/CoreServicesExtension.java:34]: Hi @jimmarino 
if we want to be consistent with the others settings just below (e.g. edc.core.retry.backoff.min...) it should even be edc.core.retry.retries.max :)
I dont have preference either, let me know which version you prefer and I'll made the change.

Comment (jimmarino) [reply core/bootstrap/src/main/java/org/eclipse/dataspaceconnector/system/CoreServicesExtension.java:34]: That seems reasonable

Comment (bscholtes1A) [reply core/bootstrap/src/main/java/org/eclipse/dataspaceconnector/system/CoreServicesExtension.java]: @jimmarino renamed as discussed

**Comment by beardyinc** [comment]
looks good to me

File: core/bootstrap/src/main/java/org/eclipse/dataspaceconnector/system/CoreServicesExtension.java (hunk 1 above)
Comment (jimmarino) [reply core/bootstrap/src/main/java/org/eclipse/dataspaceconnector/system/CoreServicesExtension.java]: @MoritzKeppler are you OK with this? If so, I'll merge

**Review by MoritzKeppler — APPROVED** [review]
fine for me, thx for cleaning up!

**Pull request merged**